### ISSUE TITLE
Success messages with bounds from termination analysis

### ISSUE THREAD

**Issue by sim642** [goblint/analyzer#1577]
Our race analysis can output "proofs" as success messages with `allglobs` enabled. This is useful for inspecting why Goblint believes the program to be correct.

We could do something similar for termination analysis: when we manage to prove a loop to be bounded and thus terminate, optionally output a success message with the bound (at least non-relationally, if not relationally). @karoliineh looked at some SV-COMP termination program where termination wasn't obvious because the loop could both increment (by 2) or decrement (by 1) the counter based on some condition.

**Comment by karoliineh** [comment]
This the example I stumbled upon was `termination-restricted-15/IntPath.c`
```c
extern int __VERIFIER_nondet_int(void);

int main() {
    int i;
    int x;
    int y;
    i = __VERIFIER_nondet_int();
    x = 0;
    y = 0;
    
    if (i > 10) {
        x = 1;
    } else {
        y = 1;
    }
    while (x == y) { }
    
    return 0;
}
```
which yields:
```
...
[Error][Analyzer][Unsound] both branches over condition 'x == y' are dead (./termination-restricted-15/IntPath.c:18:12-18:18)
SV-COMP result: true
[Warning][Analyzer] Invariant Apron: cannot convert to cil var in overflow preserving manner: term18_5_id0#1998
[Info][Witness] witness generation summary:
  total generation entries: 1
[Error][Analyzer][Unsound] Both branches dead
```

**Comment by karoliineh** [comment]
The SV-COMP task `termination-crafted-lit/KroeningSharyginaTsitovichWintersteiger-CAV2010-Fig1.c` can be used as a test case when addressing this issue.

**Comment by michael-schwarz** [comment]
I think we had such a message at some point but then removed it to be less verbose. One can probably go dig through https://github.com/goblint/analyzer/pull/1093 to find the commit where it was removed.
